### PR TITLE
Closes #126

### LINKED ISSUE #126: get_frans_github_username does not exist
Travis build error:

```
Running examples in ‘informalr-Ex.R’ failed
The error most likely occurred in:
> ### Name: get_richels_github_username
> ### Title: Get Frans's GitHub username
> ### Aliases: get_richels_github_username
> 
> ### ** Examples
> 
> get_frans_github_username()
Error in get_frans_github_username() : 
  could not find function "get_frans_github_username"
Execution halted
```

### PR DESCRIPTION
**Pull request by FrieseWoudloper**
This pull request fixes Travis build errors and closes #126 .

### CODE REVIEW

**Comment by FrieseWoudloper** [comment]
This PR wil also fix the develop and Richel branch.

File: R/get_frans_github_username.R [hunk 1]
```diff
@@ -4,6 +4,6 @@
 #' get_frans_github_username()
 #' @author Frans
 #' @export
-get_richels_github_username <- function() {
+get_frans_github_username <- function() {
```
Comment (richelbilderbeek) [inline R/get_frans_github_username.R:7]: @fransbianchi is lucky that you've just fixed his Issue :+1:

File: vignettes/hiking_app.Rmd [hunk 2]
```diff
@@ -22,7 +22,7 @@ library(informalr)
 png_filename <- tempfile(fileext = ".png")
 create_map(
   png_filename = png_filename,
-  show_bridge_openings = FALSE
+  show_bridge_openings = "no"
```
Comment (richelbilderbeek) [inline vignettes/hiking_app.Rmd:25]: Ah, our new interface, cool to see it in this place!

**Review by richelbilderbeek — APPROVED** [review]
Brilliant!

**Comment by richelbilderbeek** [comment]
Add @FrieseWoudloper: if you want, you may merge yourself, else I'll do it :+1:

**Pull request merged**